cli/cmd/install/install.go

```diff
@@ -198,6 +204,34 @@ timescaledb-single:
 		helmValues = appendPrometheusHAValues(helmValues)
 	}
 
+	if c.enableOtel {
+		helmValues = enableOtelInValues(helmValues)
+	} else {
+		e, err := helmClient.ExportValuesFieldFromChart(c.Ref, c.ConfigFile, []string{"opentelemetryOperator", "enabled"})
+		if err != nil {
+			return err
+		}
+		var ok bool
+		c.enableOtel, ok = e.(bool)
+		if !ok {
+			return fmt.Errorf("enable otel was not a bool")
+		}
+	}
+
+	if c.enableOtel {
+		// opentelemetry operator needs cert-manager as a dependency as adding cert-manager isn't good practice and
+		// not recommended by the cert-manager maintainers. We are explicitly creating cert-manager with kubectl
+		// for more details on this refer: https://github.com/jetstack/cert-manager/issues/3616
+		err := k8s.CreateK8sManifests([]string{"https://github.com/jetstack/cert-manager/releases/download/v1.5.2/cert-manager.yaml"})
+		if err != nil {
+			return err
+		}
+		fmt.Println("Installing the cert-manager...")
+		utils.ConfirmAction()
+		fmt.Println("Successfully created cert-manager")
```

Comment:
Should we maybe check that cert-manager is actually running in K8s? eg. check for pod?

Reply:
Updated, Checking for a CRD from cert-manager.